api/schedule.py

```diff
@@ -117,6 +116,10 @@ def update_schedule(schedule_id):
     Update the schedule.
     '''
     data = request.json
+    professors = get_prof_array()
+    schedule = data['schedule']
+    errors = c1alg1.validate(schedule, professors)
+    print(errors)
```

Comment:
To actually see this output in flask all print statements should have the file specified as print('...', file=stderr). If you don't do this print statements are sometimes not printed.